runtime/autoload/health/provider.vim

```diff
@@ -634,15 +634,23 @@ function! s:check_node() abort
 
   let [host, err] = provider#node#Detect()
   if empty(host)
-    call health#report_warn('Missing "neovim" npm (or yarn) package.',
+    call health#report_warn('Missing "neovim" npm (or yarn, pnpm) package.',
           \ ['Run in shell: npm install -g neovim',
           \  'Run in shell (if you use yarn): yarn global add neovim',
+          \  'Run in shell (if you use pnpm): pnpm install -g neovim',
           \  'You may disable this provider (and warning) by adding `let g:loaded_node_provider = 0` to your init.vim'])
     return
   endif
   call health#report_info('Nvim node.js host: '. host)
 
-  let manager = executable('npm') ? 'npm' : 'yarn'
+  if executable('npm')
+    let manager = 'npm'
+  elseif executable('yarn')
+    let manager = 'yarn'
+  elseif executable('pnpm')
+    let manager = 'pnpm'
+  endif
```

Comment:
the old logic defaulted to `yarn`, now `manager will be undefined if none are found, which will result in a weird error for the user.

Reply:
Maybe we should default manager to npm and then change it if we find it in the other two managers.